music/src/core/compat/global_node.ts

```diff
@@ -22,3 +22,11 @@
 export const fetch = require('node-fetch');
 export const performance = require('./performance_node');
 export const navigator = require('./navigator_node');
+
+export function isSafari(): boolean {
```

Comment:
Should it throw or should it return false, since Node isn't Safari?

Reply:
This is a tricky question - I think it's better to throw early, because checking if it's Safari is kind of meaningless in Node. But I'll make it so it just returns false... There'll be an error if people try to use audio_utils related stuff in Node, which I think is fine (eg we don't have to go far out of our way to prevent people from trying that)